specification/compute/resource-manager/Microsoft.Compute/DiskRP/stable/2023-01-02/disk.json

```diff
@@ -467,6 +467,55 @@
           "final-state-via": "location"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmssName}/disks/{diskName}/beginGetAccess": {
```

Comment:
>beginGetAccess [](http://example.com/codeflow?start=152&length=14)

could name it grantAccess?

Reply:
This new action of acquiring an access uri for a VMSS VM instance OS disk in context of VMSS is similar to an exisiting capability of acquiring access uri for a Single VM's OS Disk (Microsoft.Compute/disks -> modelled as a tracked resource in ARM), we are keeping action name symmetry -> "BeginGetAccess".
Ref: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/disks/{diskName}/beginGetAccess"